gtsfm/configs/deep_front_end.yaml

```diff
@@ -47,7 +47,7 @@ SceneOptimizer:
       min_track_len: 2
       mode:
         _target_: gtsfm.data_association.data_assoc.TriangulationParam
-        value: 0 # 0 corresponds to NO_RANSAC
+        value: 1 # 0 corresponds to NO_RANSAC
```

Comment:
why change in this PR?

Reply:
same comment -- let's leave as 0 here

Reply:
Yes, was testing something; will change back

Reply:
fixed